src/Generator/PropertyGenerator.php

```diff
@@ -67,15 +64,17 @@ public static function fromReflection(PropertyReflection $reflectionProperty)
             $property->omitDefaultValue = true;
         }
 
-        if ($reflectionProperty->getDocComment() != '') {
-            $property->setDocBlock(DocBlockGenerator::fromReflection($reflectionProperty->getDocBlock()));
+        $docBlock = $reflectionProperty->getDocBlock();
+
+        if ($docBlock) {
```

Comment:
Same here regarding explicitly checking for absence of false or the class instance. Your choice.

Reply:
Yeah, rolling with a simple conditional, since this is `false|object`